creator-node/src/ipfsClient.js

```diff
@@ -0,0 +1,25 @@
+const ipfsClient = require('ipfs-http-client')
+const ipfsClientLatest = require('ipfs-http-client-latest')
```

Comment:
what is the point of using this? why not just use ipfsClient, that is the official release - https://www.npmjs.com/package/ipfs-http-client

Reply:
ok i see that this is what was already here -  no need to fix this one!